seed/src/lib.rs

```diff
@@ -135,15 +137,15 @@ impl Default for NodeConfig {
 /// Seed node instance.
 pub struct Node {
     /// Sender end of user requests.
-    handle: chan::UnboundedSender<Request>,
+    handle: mpsc::Sender<Request>,
     /// Receiver end of user requests.
-    requests: chan::UnboundedReceiver<Request>,
+    requests: mpsc::Receiver<Request>,
 }
 
 impl Node {
     /// Create a new seed node.
     pub fn new() -> Result<Self, Error> {
-        let (handle, requests) = chan::unbounded::<Request>();
+        let (handle, requests) = mpsc::channel::<Request>(64);
```

Comment:
Is there any particular logic to choosing this size for the buffer?

Reply:
Power of 2, as meaningful as 42. Magic constant until a better heuristic has been found or it is surfaced as config param.